.travis.yml

```diff
@@ -1,34 +0,0 @@
-language: groovy
```

Comment:
If the travis file is present in a branch it will run the CI automatically. The thing is at the moment the build is failling randomly for no reason. Everything works in local, but it's failling on travis. I didn't want to either fix the travis or run a lot of failling builds. So I deleted the file. Simon propose to move to Jenkins too.